### ISSUE TITLE
Tapping on the "screen" during a game brings up the file selection

### ISSUE THREAD

**Issue by bhdouglass** [RyanPattison/Gearboy#15]
Tapping on the "screen" during a game causes the file manager to popup again. Which is fine if you want to switch games, but hitting it accidentally will cause the current game to stop working. It might make sense to move the "load rom" functionality into settings menu during gameplay.

**Comment by RyanPattison** [comment]
This was annoying me too. For now at least, I made the tappable area of the screen smaller as a temporary fix.

**Comment by RyanPattison** [comment]
This shouldn't cause the game to stop working. When the file manager is closed it cancels the import of the ROM and the game should resume, correct?

**Comment by bhdouglass** [comment]
Yeah, if you specifically hit the cancel button in the filemanager the game resumes, but if you switch back to the game without canceling the filemanager the game freezes.

**Comment by RyanPattison** [comment]
Thanks!  This is related to #18 and maybe even #16, sorry I missed that point earlier.

**Comment by bhdouglass** [comment]
No worries!
